webapp/maps_heatmaps/firebase.json

```diff
@@ -0,0 +1,16 @@
+{
+  "hosting": {
+    "public": "public",
+    "ignore": [
+      "firebase.json",
+      "**/.*",
+      "**/node_modules/**"
+    ],
+    "rewrites": [
+      {
+        "source": "**",
+        "destination": "/index.html"
+      }
+    ]
+  }
+}
```

Comment:
Can you add a docstring at the top of the file with a URL pointer on why we need this file and what it does?

Reply:
We added a docstring as another key-value pair in the data of the file. Is that ok or should we find another way? we read that it is not possible to add doctrings in json files.
https://www.freecodecamp.org/news/json-comment-example-how-to-comment-in-json-files/